src/components/molecules/Banner/Banner.tsx

```diff
@@ -50,12 +50,16 @@ export const Banner: React.FC<BannerProps> = (props) => {
           </Button>
         </Col>
         <Col xs={12} md={mobile ? 12 : 6}>
-          <img
-            src={img.src}
-            alt={img.alt}
-            title={img.title}
-            className={"banner-img"}
-          />
+          {React.isValidElement(img) ? (
+            <React.Fragment>{img}</React.Fragment>
+          ) : (
+            <img
+              className={"banner-img"}
+              src={(img as BannerImgProps).src}
+              alt={(img as BannerImgProps).alt}
+              title={(img as BannerImgProps).title}
+            />
+          )}
```

Comment:
if you know another better / simpler solution to this notation, let me know, I will improve it

Reply:
it's ok i think